UnityProject/Assets/Scripts/Robotics/Bots/InteractableBot.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using UnityEngine;
+using Mirror;
+using Systems.MobAIs;
+
+namespace Robotics
+{
+	public class InteractableBot : NetworkBehaviour, IPredictedCheckedInteractable<HandApply>
+	{
+		HandApply interaction;
+
+		[SerializeField] private SpriteRenderer SPRITE_RENDERER;
```

Comment:
should use ``SpriteHandler`` instead of ``SpriteRenderer`` directly. The formas has the advantage of networking the sprite changes. As it is right now, sprites won't change on all connected clients.